littlechef/runner.py

```diff
@@ -45,6 +45,7 @@
 
 __testing__ = False
 
+env.berksfile_cookbooks_directory=""
```

Comment:
There should be spaces to both sides of the equality sign. Also a better place to put this statement would be further up, for example right below `env.node_work_path = littlechef.node_work_path`